core/types/gateway_metadata.go

```diff
@@ -5,14 +5,16 @@ package types
 
 // GatewayMetadata contains metadata for each gateway that received a message
 type GatewayMetadata struct {
```

Comment:
What about the Frequency Offset and the other RSSI's?

Reply:
According to our contact at Semtech those fields are not needed at this point.